apps/dashboard/src/app/company/[slug]/company-page.tsx

```diff
@@ -1,5 +1,5 @@
 import { GenericTable } from "@/components/GenericTable"
-import type { Event } from "@dotkomonline/types"
+import type { EventDetail } from "@dotkomonline/types"
```

Comment:
Prefer to rename this to EventWithAttendance probably, EventDetail is not very descriptive